.gitattributes

```diff
@@ -1,3 +1,3 @@
 # text files must be lf for golden file tests to work
-*.txt eol=lf
-*.json eol=lf
+* text eol=lf
```

Comment:
I had back then `* text=auto eol=lf` not sure why, but it was passing for me: https://github.com/microsoft/playwright/pull/7987

Reply:
Yep, reading the docs it looks like text=auto is what we want!